consensus/src/tests/consensus_tests.rs

```diff
@@ -91,11 +91,11 @@ async fn commit_one() {
         .collect::<BTreeSet<_>>();
     let (mut certificates, next_parents) = make_certificates(1, 2, &genesis, &keys);
 
-    // Make two certificate (f+1) with round 3 to trigger the commits.
+    // Make one certificate with round 3 to trigger the commits.
```

Comment:
The new certificate is in round 3, which is in this case steady state wave 2 (ceil(3/2)). Since this certificate is entering a new wave (was in steady state wave 1 previously), it should trigger try_steady_commit and commit the leader of steady state wave 1 (line 68 of Algorithm 3 in the Bullshark paper).